scipy/integrate/quadpack.py

```diff
@@ -560,6 +560,18 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
     romb : integrator for sampled data
     scipy.special : for coefficients and roots of orthogonal polynomials
 
+    Examples
+    --------
+
+    Calculate :math: double integration x*y**2
```

Comment:
Please wrap the expression into double backticks so that it renders as code.

Reply:
so instead of this line I am replacing it with "Returns  double integration of ``x*y**2``  "(without double quotes) . Is this correct @ev-br